distribution/src/site/content/tabular-parameters.html

```diff
@@ -385,6 +385,32 @@ <h2>Modifying content of tabular parameter</h2>
 the scenario writer. The difference is in the <b>Examples:</b> keyword,
 which is only present for parametrised scenarios.</span>
 
+<h2>Working with self-referencing tables</h2>
+
+<p>By default, JBehave does not support self-referencing relationship in example tables (when a column value in a table relates to another column in the same table).
```

Comment:
(when a column value in a table relates to another column in the same table).
IMHO it will be clearer if we replace table mentions with row ones